### DISCUSSION TITLE
Mac M1

### DISCUSSION THREAD

**Discussion by marklacas** [tabemann/zeptoforth#17] [Ideas]
Have you considered porting to the Mac M1.
I would use and extend it.
Thanks,
-ml

**Comment by tabemann** [comment]
I should note that zeptoforth is for systems that run the ARM Thumb or Thumb-2 instruction sets and which are 32-bit and the Apple M SoC's are specifically 64-bit, with no support for 32 bit and no support for the Thumb or Thumb-2 instruction sets, which effectively rules out support by zeptoforth without a complete rewrite.

**Comment by marklacas** [comment]
Ok, thanks for responding. 

That’s too bad.

Cheers,
-ml

**Comment by tabemann** [comment]
One thing, however, is that potentially one might be able to run zeptoforth under QEMU in the future.

**Reply by marklacas** [reply]
Cool!
